1786 | Proposition pages HCM

chris-lawton opened the pull request:
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1230241786)

### Description of Changes Made
- Add hover states for in-page nav, quote slider and pull quote
- Add a light background behind client logos so they remain visible
- Lower the underline on keypoints so it doesn't overlap the bottom border

### How to Test
- Visit a proposition page locally - `http://localhost:8000/digital-products/`
- Enable forced-colors: active for HCM and prefers-color-scheme for light/dark mode

### Screenshots

<details>
  <summary>Expand to see more</summary>

![Screenshot 2023-07-28 at 12 53 21](https://github.com/torchbox/wagtail-torchbox/assets/31627284/74ac886b-f958-4a32-a51e-f03cd962ba44)

![Screenshot 2023-07-28 at 12 53 33](https://github.com/torchbox/wagtail-torchbox/assets/31627284/c32035c4-85e5-4144-87f9-e80aa6b4a489)

![Screenshot 2023-07-28 at 12 53 39](https://github.com/torchbox/wagtail-torchbox/assets/31627284/c09d248a-63c3-4829-8b86-a89eb2b5f3dd)


</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [ ] Consider adding unit tests, especially for bug fixes. If you don't, tell us why.
- [ ] Tests and linting passes.
- [ ] Consider updating documentation. If you don't, tell us why.
- [ ] If relevant, list the environments / browsers in which you tested your changes.

helenb commented:
![Screenshot 2023-07-31 at 10 41 52](https://github.com/torchbox/wagtail-torchbox/assets/771869/89a62e9b-17e2-44e9-966a-f38575e8a1ce)

https://github.com/torchbox/wagtail-torchbox/assets/771869/546a4cf6-92c1-4513-b38a-3e5af78ea026

chris-lawton commented:
Thanks Helen. I've addressed your feedback all on this MR. I've removed the increase in logo size on hover and instead replaced with an underline on the 'Read case study' text so it's consistent with the other hover states in HCM. I've also moved the white bg to the image rather than the entire link so the text is more visible.
